### PR TITLE
First iteration of the main page for content creators for plc - lets …

### PR DESCRIPTION
**Pull request by mehalshah**
…them view and create courses and modules

This gets us closer to the spec in progress here - https://docs.google.com/document/d/1_ByhpochtiGsHOF6hFkUgezf9wZxa4gI4UAHmClGezk/edit

Subsequent PRs will remove the original scaffolding code

### CODE REVIEW

File: code-studio/src/css/plc/content_creation [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.outlinedSection {
+  border: 1px solid;
+  padding: 1em;
+}
+
+.newLink {
+  margin-top: 24px;
+}
```
Comment (tanyaparker) [inline code-studio/src/css/plc/content_creation]: nit: EOF line

File: dashboard/app/views/plc/content_creator/show_courses_and_modules.html.haml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+- content_for(:head) do
+  = stylesheet_link_tag 'css/levelbuilder', media: 'all'
+
+%h1 Professional Learning Content
+
+.outlinedSection
+  %h4 Courses
+  -@courses.each do |course|
+    =link_to course.name, course
```
Comment (tanyaparker) [inline dashboard/app/views/plc/content_creator/show_courses_and_modules.html.haml]: nit: consistency that there is a space between the - or = before the code. same for learning_modules

```
- @courses.each do |course|
  = link_to course.name, course
```

File: dashboard/app/views/plc/content_creator/show_courses_and_modules.html.haml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+%br
+
+.outlinedSection
+  %h4 Learning Modules
+  -@learning_modules.each do |learning_module|
+    =link_to learning_module.name, learning_module
+    %br
+  .newLink
+    = link_to 'Add new learning module', new_plc_learning_module_path
```
Comment (tanyaparker) [inline dashboard/app/views/plc/content_creator/show_courses_and_modules.html.haml]: nit: EOF line

File: code-studio/src/css/plc/content_creation [hunk 4]
```diff
@@ -0,0 +1,8 @@
+.outlinedSection {
```
Comment (tanyaparker) [inline code-studio/src/css/plc/content_creation]: Use dash case for class names

**Comment by tanyaparker** [comment]
In case you don't already have this logged, when I created a new module I got two different kinds of successful notification.

image

File: code-studio/src/css/plc/content_creation (hunk 4 above)
Comment (tanyaparker) [inline code-studio/src/css/plc/content_creation]: I think the file name should also be dash case. I'm not sure why the existing .scss in code-studio uses underscores, but we use dash case for all our other css and scss files.

**Comment by tanyaparker** [comment]
Is there special setup for code-studio scss? In my local the classes aren't being applied.

image

**Comment by mehalshah** [comment]
The extra notice is a relic of the scaffold generation - I'll take it all out

npm run build should build the classes properly

Will take care of the other suggestions

**Comment by tanyaparker** [comment]
I didn't see it in the spec, but I think the edit module page should have a link back to "home" which I think is the course list?

**Comment by tanyaparker** [comment]
I need to fix my code-studio build but otherwise the skeleton lgtm

**Comment by mehalshah** [comment]
I'm changing the links to direct users back to the show_content_creator page

**Comment by mehalshah** [comment]
Or rather - will do so with subsequent PR

**Pull request merged**